nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

```diff
@@ -12,6 +12,33 @@ These release notes track the development of the {secondary-scheduler-operator-f
 
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
 
+[id="secondary-scheduler-operator-release-notes-1.1.1"]
+== Release notes for {secondary-scheduler-operator-full} 1.1.1
+
+Issued: 2023-3-6
+
+The following advisory is available for the {secondary-scheduler-operator-full} 1.1.1:
+
+* link:https://access.redhat.com/errata/RHSA-2023:0584[RHSA-2023:0584]
+
+////
+// No enhancements in this release
+[id="secondary-scheduler-operator-1.1.1-new-features-and-enhancements"]
+=== New features and enhancements
```

Comment:
Purely a suggestion. Is it better to have the section and explicitly state that there are no features or enhancements in this release, rather than removing section?

Reply:
Good question. I checked other release notes sections and they don't seem to list them if there aren't any items for that section.